### DISCUSSION TITLE
Bagaimana Cara Setting Tampilan Shiny Apps?

### DISCUSSION THREAD

**Discussion by jeremiaww** [teamalgoritma/community#100] [Q&A] [answered]
Hello,

Izin bertanya bagaimana cara setting tampilan shiny apps di handphone? Agar tampilannya bisa menyesuaikan rapih di handphone (android).

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/99194718/168188920-cbe3a616-ab3b-4978-9010-2d55210fde56.png">

Terima kasih

**Comment by kevbow** [comment, marked as answer]
Halo Pak Jeremia,
Untuk pengaturan dashboard shiny pada mobile apps bisa menggunakan package lain yaitu "shinyMobile". Informasi lebih lengkap mengenai package ini dapat dilihat pada link berikut: https://rinterface.github.io/shinyMobile/.
Terima kasih dan semoga membantu.

**Reply by jeremiaww** [reply]
Terima kasih mas, nanti akan saya coba. Thanks
